test/integration/integration_test.go

```diff
@@ -71,10 +72,68 @@ func getDefaultContext() (context.Context, context.CancelFunc) {
 
 // TestInjectionBuild tests the build where we inject files to assemble script.
 func TestInjectionBuild(t *testing.T) {
-	integration(t).exerciseInjectionBuild(TagCleanBuild, FakeBuilderImage, []string{
```

Comment:
With this change I noticed you don't call `setup` anymore, as `integration(...)` does. Is there any particular reason for that?

I'm doing some improvements to integration tests that rely on the fact that I can write common initialization to all tests, and the fact that they call `integration` helps with that.